lib/routes/api/speech-credentials.js

```diff
@@ -221,11 +222,21 @@ router.post('/', async(req, res) => {
       }
     }
 
+    // Check if vendor and label is already used for account or SP
+    if (label) {
+      const existingSpeech = await SpeechCredential.isAvailableVendorAndLabel(
+        service_provider_sid, account_sid, vendor, label);
+      if (existingSpeech.length > 0) {
+        throw new DbErrorUnprocessableRequest(`Label ${label} is used for vendor ${vendor}`);
```

Comment:
please change error to "Label ${label} is already in use for another speech credential"